### ISSUE TITLE
Unable to add content

### ISSUE THREAD

**Issue by CaseOf** [etesync/server#89]
Hello,
I am running my own instance but I cant add content on it.
uvicorn gives me errors 500 in logs:
```
INFO:     IP-Address - "POST /api/v1/collection/list_multi/?stoken=CWVtWa_N94Cza4T6GsA1D-A7OqaW2DUM&limit=40 HTTP/1.1" 200 OK
INFO:     IP-Address - "POST /api/v1/collection/z6f9SgbnK-nQAWCqCN5GDLh7ZQ6mNZ7C/item/batch/ HTTP/1.1" 500 Internal Server Error
INFO:     IP-Address - "POST /api/v1/collection/list_multi/?stoken=CWVtWa_N94Cza4T6GsA1D-A7OqaW2DUM HTTP/1.1" 200 OK
INFO:     IP-Address - "POST /api/v1/collection/lDRVHNSt3Fnl8oLeBc2oJfGh3zYlPMYF/item/batch/ HTTP/1.1" 500 Internal Server Error
INFO:     IP-Address - "OPTIONS /api/v1/collection/j-SvIpfCVdIltKBLZcAoFl2B4LN0y1_L/item/batch/ HTTP/1.1" 200 OK
INFO:     IP-Address - "POST /api/v1/collection/j-SvIpfCVdIltKBLZcAoFl2B4LN0y1_L/item/batch/ HTTP/1.1" 500 Internal Server Error
INFO:     IP-Address - "POST /api/v1/collection/list_multi/?stoken=CWVtWa_N94Cza4T6GsA1D-A7OqaW2DUM HTTP/1.1" 200 OK
INFO:     IP-Address - "POST /api/v1/collection/lDRVHNSt3Fnl8oLeBc2oJfGh3zYlPMYF/item/batch/ HTTP/1.1" 500 Internal Server Error
INFO:     IP-Address - "POST /api/v1/collection/list_multi/?stoken=CWVtWa_N94Cza4T6GsA1D-A7OqaW2DUM HTTP/1.1" 200 OK
INFO:     IP-Address - "POST /api/v1/collection/lDRVHNSt3Fnl8oLeBc2oJfGh3zYlPMYF/item/batch/ HTTP/1.1" 500 Internal Server Error
```
I can access my account from the Android app and pim.etesync.com but I cant add anything.

**Comment by tasn** [comment]
There's no info here. We would need to see what the actual errors are. How are you running the server itself? Can you see the debug output?

**Comment by CaseOf** [comment]
Well, as talked about it on IRC, it was a config.ini issue, I did not set the media path, but the comment was confusing to not touch things unless you know what you are doing, that you fixed in 21e5382fc43b57edabf322e018eb746dc717bb37.

**Comment by elig0n** [comment]
I had a similar problem when running uvicorn as user `etesync`. Giving the user and group write permissions to the `/var/lib/etebase-server` directory and its files solved it. 

Regarding @CaseOf solution, I am not sure what paths should be set in those variables and I haven't found any exact information or documentation about them in the wiki or in the Issues.
